examples/src/retrievers/pinecone_hybrid.ts

```diff
@@ -0,0 +1,30 @@
+import { PineconeClient } from "@pinecone-database/pinecone";
+import { BertTokenizer } from "bert-tokenizer";
+
+import { OpenAIEmbeddings } from "langchain/embeddings";
+import { PineconeHybridSearchRetriever } from "langchain/retrievers";
+
+export const run = async () => {
+  const client = new PineconeClient();
+
+  await client.init({
+    environment: process.env.PINECONE_ENVIRONMENT!,
+    apiKey: process.env.PINECONE_API_KEY!,
+  });
+
+  const embeddings = new OpenAIEmbeddings();
+  const pineconeIndex = client.Index(process.env.PINECONE_INDEX!);
+
+  const tokenizer = new BertTokenizer(undefined, true, 512);
```

Comment:
Consider using `js-tiktoken` instead, which should match the behavior of `OpenAIEmbeddings`